Esto es una prueba de pull request

issacvega opened the pull request.

issacvega committed:
Esto es una prueba de pull request

issacvega committed:
Add files via upload

Codigo para calcular las energias de una particula en una caja usando el metodo variacional. Ejercicio 1

lazarusA reviewed (changes requested):
Se ve bien, en principio. Sin embargo, creo que deberías incluir un notebook donde especifiques el numero de funciones, y las funciones(el ejemplo en acción). Otra cosa, eventualmente también nos gustaría tener una función general en la que el Hamiltoniano se especificará a la hora de correr dicha función. Entonces, hasta que tengas esos cambios será posible aceptar tu propuesta. Saludos.

issacvega committed:
Add files via upload

Programa para resolver sistemas con el metodo variacional para diferentes hailtonianos.

issacvega committed:
Delete Metodo_Variacional.ipynb

issacvega committed:
Delete pruebaissac.txt

issacvega committed:
Add files via upload

Programa para resolver sistemas con el metodo variacional y diferentes hamiltonianos.

issacvega committed:
Add files via upload

Metodo de Perturbaciones

issacvega committed:
Add files via upload

Simetrias
